processors/games/HungerGames.js

```diff
@@ -1,8 +1,221 @@
+/* eslint-disable camelcase */
 /*
-* Blitz Survival Games
+ * Blitz Survival Games
  */
+
+const {
+  getRatio,
+  getWeeklyStat,
+  getMonthlyStat,
+} = require('../../util/utility');
+
 module.exports = ({
-  coins = 0,
-}) => ({
+  // General
   coins,
-});
+  deaths,
+  kills,
+  wins,
+  wins_teams,
+  damage_taken,
```

Comment:
Looks like you forgot to return some values, which causes linting to [fail](https://travis-ci.org/slothpixel/core/builds/438949874?utm_source=github_status&utm_medium=notification).